src/cli.zig

```diff
@@ -511,7 +511,14 @@ pub const Arguments = struct {
         opts.env_files = args.options("--env-file");
         opts.extension_order = args.options("--extension-order");
 
-        ctx.passthrough = args.remaining();
+        {
+            const positionals = args.positionals();
+            const remaining = args.remaining();
+            var list = try std.ArrayList(string).initCapacity(allocator, positionals.len + remaining.len);
+            list.appendSliceAssumeCapacity(positionals);
+            list.appendSliceAssumeCapacity(remaining);
+            ctx.passthrough = list.items;
+        }
```

Comment:
What's the block `{`.... `}` for here?

Reply:
zig has strict no-shadowing rules so i did the local block here to prevent `list` from leaking into the main scope since its not needed once assigned to `ctx.passthrough`